query/transaction.ts

```diff
@@ -386,9 +418,34 @@ export class Transaction {
     query: string,
     args?: QueryArguments,
   ): Promise<QueryArrayResult<T>>;
+  /**
+   * Use the configuration object for more advance options to execute the query
+   *
+   * ```ts
+   * import { Client } from "./client.ts";
```

Comment:
```suggestion
   * import { Client } from "https://deno.land/x/postgres/mod.ts";
```
Ditto for all other snippets. This should fix your errors.

Reply:
even when it is in a comment, the linter thinks it is a real path? That was just an example code

Reply:
Yep 🙂